config/module.config.php

```diff
@@ -2,11 +2,11 @@
 return [
     'service_manager' => [
         'invokables' => [
-            'Dash\Router\Http\Parser\ParserManager' => 'Dash\Router\Http\Parser\ParserManager',
-            'Dash\Router\Http\Route\RouteManager' => 'Dash\Router\Http\Route\RouteManager',
+            Dash\Router\Parser\ParserManager::class => Dash\Router\Parser\ParserManager::class,
```

Comment:
It feels ridiculous though when the use statements have the same amount of line numbers than the rest of the config. Wonder if @Ocramius has any thoughts on this.